package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "bash scripts/build.sh --fetch playbook.yml",
     "build:no-fetch": "bash scripts/build.sh playbook.yml",
-    "build:local": "bash scripts/build.sh playbook-local-only.yml",
+    "build:local": "yarn prepare && chmod +x scripts/build.sh && scripts/build.sh playbook-local-only.yml",
```

Comment:
Is there a need to add +x instead of using `bash` to run the script?  Note the other scripts above and below this line still use `bash`.

Reply:
I ran into issue with script being executable so added this for my testing, reverted this change.